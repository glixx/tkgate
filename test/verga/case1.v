//: beginnetlist top
module top;
  integer i,q;

  initial
    begin
	q = 1 + 1;
	for (i = 0;i < 6;i = i + 1)
	  case (i)
		8'h1: $display("one");
		4'h2: $display("two");
		(q+1): $display("three");
		4: $display("four");
		default: $display("default");
	  endcase
    end

endmodule
//:endnetlist top
