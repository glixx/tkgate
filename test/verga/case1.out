default
one
two
three
four
default
